packages/devtools/src/commands/closeWindow.ts

```diff
@@ -27,6 +32,10 @@ export default async function closeWindow () {
     }
 
     const newPage = this.windows.get(this.currentWindowHandle)
+    if (!newPage) {
```

Comment:
Maybe it makes sense to have a method that would get page or throw instead of copy pasting?

Reply:
this is not optimal, I would like to postpone this into a separate PR though

Reply:
fixed